### ISSUE TITLE
The Count given is zero

### ISSUE THREAD

**Issue by milu234** [mathdroid/covid-19-api#186]

**Comment by Lms24** [comment]
Experiencing the same issue. Just to add some details, when e.g. GETting `api/countries/de/`, the returned result is 0 for recovered, confirmed and deaths

**Comment by leonlovett** [comment]
Same here. Also getting 0 for confirmed, recovered and deaths on `/api`

**Comment by milu234** [comment]
Anybdy can contact them or something

**Comment by milu234** [comment]
Atleast Let us know when it will be updated Because my website is live 
https://coronatracker-1cf6e.web.app/

**Comment by milu234** [comment]
Even the Count given is not proper 

But  I really like the API Please try to fix it as soon as possible

**Comment by milu234** [comment]
> Same here. Also getting 0 for confirmed, recovered and deaths on `/api`

Recovered and deaths are added now

**Comment by leonlovett** [comment]
Looks like it's an issue with the data at JHU and not necessarily a problem with the API.
https://coronavirus.jhu.edu/map.html

**Comment by ravik73** [comment]
Any upadte on when the confirmed will be fixed?

**Comment by milu234** [comment]
Well I guess we also at our end have to go for maintenance also for a while 

Actually nation wide lock down was just extended I think so they are updating

**Comment by lividavi19** [comment]
Same with me, I tried below endpoint and it gives 0 cases.

https://covid19.mathdro.id/api/countries/TZ

**Comment by lividavi19** [comment]
Let's wait for the update maybe..

**Comment by rk4bir** [comment]
Same here for /countries/BD is zero. Please this asap. My Android app is live 😑😑

**Comment by milu234** [comment]
Yeah I think so It will take some time till then just make a pop up of maintaineance

**Comment by YogaSakti** [comment]
JHU data is currently under maintenance.

**Comment by milu234** [comment]
The  apis are resumed 
Check out this 
https://coronatracker-1cf6e.web.app/

**Comment by milu234** [comment]
Again it is fluctuating

**Comment by Lms24** [comment]
There's something wrong at JHU. The whole thing probably doesn't have to do anything with the API. Look at their confirmed infection curve, it shows decreases on april 22 and 24, which is obivously impossible. But yeah, since I'm relying on JHU to track worldwide data for my own application, I really hope that they'll fix it soon or else I'll lose this day in my dataset...

**Comment by milu234** [comment]
> There's something wrong at JHU. The whole thing probably doesn't have to do anything with the API. Look at their confirmed infection curve, it shows decreases on april 22 and 24, which is obivously impossible. But yeah, since I'm relying on JHU to track worldwide data for my own application, I really hope that they'll fix it soon or else I'll lose this day in my dataset...

I think so they fixed it but still Ecuador Country Count shows zero
